fractal-puppy/Engine
# Changed LC slot icon

The Left Click is used for moving and using basic attack, you can not equip any skill. That is why I changed the icon.
![Anotación 2019-10-20 170434](https://user-images.githubusercontent.com/17391875/67161580-3d26bb00-f35c-11e9-829e-e81aa342e16a.jpg)

Also, I added the audio prefabs to level 2 so it doesn't crash.

**To test:**
- Open the level 1 scene and check that the icon is there and you can not equip anything.
- Same for leve 2.
